README.md

```diff
@@ -217,6 +217,18 @@ bbb_greenlight_office365:
     hd:
 ```
 
+#### OPENID CONNECT
+
+For in-depth steps on setting up a OpenID Connect Login Provider, see: <https://docs.bigbluebutton.org/greenlight/gl-config.html#openid-connect>
```

Comment:
There does not seem to be a OpenID Connect part in the Greenlight docs?

Reply:
No, not yet. See: https://github.com/bigbluebutton/bigbluebutton.github.io/issues/287